src/js/core/widget/core/Popup.js

```diff
@@ -644,6 +648,23 @@
 				eventUtils.on(self._ui.page, "pagebeforehide", self, false);
 				eventUtils.on(window, "resize", self, false);
 				eventUtils.on(document, "vclick", self, false);
+
+				self._ui.content.addEventListener("scroll", function () {
```

Comment:
Please exclude this anonymous function to named function, eg. `prototype._onScroll = function() { `

Reply:
Second thing,
remove this listener in `_unbindEvents` method